test/promise/simple.js

```diff
@@ -49,4 +49,48 @@ describe('Promise', function () {
       Promise.all.call(StealingPromiseConstructor, iterable);
     });
   });
+
+  specify('resolve with a thenable calls it once', function () {
```

Comment:
Both of these are an async test, so it needs the `done` argument - ie, `Promise.all([a, b]).then(done, done)` at the bottom of the tests.